pkgs/by-name/em/emacsclient-commands/package.nix

```
@@ -0,0 +1,28 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, buildGoModule, runCommandLocal }:
+
+buildGoModule rec {
+  pname = "emacsclient-commands";
+  version = "0.0.0-unstable-2023-09-22";
```

```suggestion
  version = "unstable-2023-09-22";
```
I think we should leave out the version since the repo doesn't have any yet.

---

For future reference - the current convention is `0-unstable-DATE` if there are no releases/tags